docs/stories/stories.md

```diff
@@ -0,0 +1,79 @@
+# COGS User Stories
+
+## Roles:
+User
+Project owner
+Principal Investigator
+Project Authoriser
+Attribution Authoriser
+RSE
+System Administrator
+
+## Stories
+### Project creation
+- Users with valid, non-student institutional email addresses can create new projects as the project owner
+- Users with student or non-institutional email address cannot create new projects
+- New projects are created with a default priority level and are not active
```

Comment:
All projects must be approved before they are assigned a project code and made real.  The only policy based concept of prority that exists at the moment is that of the Category that a project is assigned to (as mentioned in the business plan), and this must be set by SCW person as part of the approval process.  The Category is then used to influence the actual compute allocation.

Reply:
Apologies for my wording again, by "created" again I meant in the sense of a "creating a database record" and by "priority" I was referring to category. This wasn't raised by me, but I think a sensible default value in the submission form isn't a terrible idea and, in my view, would be enough to fulfil the requirement.